packages/etherscan/src/lib/etherscan.ts

```diff
@@ -149,117 +193,321 @@ async function getSourceCode(call: Etherscan, address: string) {
 // TRANSACTIONS //
 //////////////////
 
-function getStatus(call: Etherscan, txhash: string) {
-  return call<ExecutionStatusResult>({ module: 'transaction', action: 'getstatus', txhash });
+async function getStatus(call: Etherscan, txhash: string) {
+  const res = await call<ExecutionStatusResult>({ module: 'transaction', action: 'getstatus', txhash });
+  return {
+    isError: toBoolean(res.isError),
+    errDescription: res.errDescription
+  }
 }
 
-function getTxReceiptStatus(call: Etherscan, txhash: string) {
-  return call<StatusResult>({ module: 'transaction', action: 'gettxreceiptstatus', txhash });
+async function getTxReceiptStatus(call: Etherscan, txhash: string) {
+  const res = await call<StatusResult>({ module: 'transaction', action: 'gettxreceiptstatus', txhash });
+  return {
+    status: toBoolean(res.status),
+  }
 }
 
 ////////////
 // BLOCKS //
 ////////////
 
-function getBlockReward(call: Etherscan, blockno: number) {
-  return call<BlockRewardResponse>({ module: 'block', action: 'getblockreward', blockno });
+async function getBlockReward(call: Etherscan, blockno: number) {
+  const res = await call<BlockRewardResponse>({ module: 'block', action: 'getblockreward', blockno });
+  return {
+    blockNumber: toNumber(res.blockNumber),
+    timeStamp: res.timeStamp,
+    blockMiner: res.blockMiner,
+    blockReward: toBigNumber(res.blockReward),
+    uncles: res.uncles,
+    uncleInclusionReward: toNumber(res.uncleInclusionReward)
+  }
```

Comment:
To avoid to repeat every property, you can use a spread operator and then change the properties.
Also, you can update `timeStamp` to `Date` no ?
```suggestion
  return {
    ...res,
    blockNumber: toNumber(res.blockNumber),
    uncleInclusionReward: toNumber(res.uncleInclusionReward),
    blockReward: toBigNumber(res.blockReward),
    timeStamp: toDate(res.timeStamp),
  }
```